projects/1.16/assets/tinkers-archery/tinkersarchery/lang/zh_cn.json

```
@@ -0,0 +1,247 @@
+  "modifier.tinkersarchery.highlander.description": "箭在高处能射得更远",
+  "modifier.tinkersarchery.highlander.flavor": "直到我的膝盖中了一箭",
+  "modifier.tinkersarchery.launching": "充能",
+  "modifier.tinkersarchery.launching.1": "充能Ⅰ",
+  "modifier.tinkersarchery.launching.2": "充能Ⅱ",
+  "modifier.tinkersarchery.launching.3": "充能Ⅲ",
+  "modifier.tinkersarchery.launching.4": "充能Ⅳ",
+  "modifier.tinkersarchery.launching.5": "充能Ⅴ",
+  "modifier.tinkersarchery.launching.description": "箭射出的速度更快",
+  "modifier.tinkersarchery.launching.flavor": "跑！巴里！跑！",
+  "modifier.tinkersarchery.looping_trajectory": "环形轨迹",
+  "modifier.tinkersarchery.looping_trajectory.description": "箭在空中盘旋一圈，然后直线飞行，最后再次盘旋",
+  "modifier.tinkersarchery.looping_trajectory.flavor": "哇哦... 哇哦",
+  "modifier.tinkersarchery.multishot": "多重射击",
+  "modifier.tinkersarchery.multishot.description": "弓射击时附加更多箭",
+  "modifier.tinkersarchery.multishot.flavor": "现在我们有三倍火力！",
+  "modifier.tinkersarchery.pinpointer": "精准",
+  "modifier.tinkersarchery.pinpointer.1": "精准Ⅰ",
+  "modifier.tinkersarchery.pinpointer.2": "精准Ⅱ",
+  "modifier.tinkersarchery.pinpointer.3": "精准Ⅲ",
+  "modifier.tinkersarchery.pinpointer.4": "精准Ⅳ",
+  "modifier.tinkersarchery.pinpointer.5": "比精准Ⅳ更精准",
+  "modifier.tinkersarchery.pinpointer.description": "弓射击时更精准",
+  "modifier.tinkersarchery.pinpointer.flavor": "精灵王子对此一无所知",
+  "modifier.tinkersarchery.power": "强力",
+  "modifier.tinkersarchery.power.1": "强力Ⅰ",
+  "modifier.tinkersarchery.power.2": "强力Ⅱ",
+  "modifier.tinkersarchery.power.3": "强力Ⅲ",
+  "modifier.tinkersarchery.power.4": "强力Ⅳ",
+  "modifier.tinkersarchery.power.5": "强力Ⅴ",
+  "modifier.tinkersarchery.power.description": "箭头造成更多伤害",
+  "modifier.tinkersarchery.power.flavor": "怎么做到的?",
+  "modifier.tinkersarchery.slicing": "切割",
+  "modifier.tinkersarchery.slicing.description": "箭总是危险的",
+  "modifier.tinkersarchery.slicing.flavor": "削铁如泥",
+  "modifier.tinkersarchery.superslime": "超级史莱姆",
+  "modifier.tinkersarchery.superslime.description": "你弓上的黏液覆层越多，你拉弓就越快，箭也飞得越远",
+  "modifier.tinkersarchery.superslime.draw_speed": "超级史莱姆拉弓速度",
+  "modifier.tinkersarchery.superslime.elasticity": "超级史莱姆拉弓磅数",
+  "modifier.tinkersarchery.superslime.flavor": "Duang Duang的",
+  "modifier.tinkersarchery.swiftstrike": "移动射击",
+  "modifier.tinkersarchery.swiftstrike.attack_damage": "移动伤害",
+  "modifier.tinkersarchery.swiftstrike.description": "移动速度加快会造成更多伤害",
+  "modifier.tinkersarchery.swiftstrike.flavor": "放轻松~",
+  "modifier.tinkersarchery.twirling_trajectory": "转动轨迹",
+  "modifier.tinkersarchery.twirling_trajectory.description": "箭在空中旋转",
+  "modifier.tinkersarchery.twirling_trajectory.flavor": "做一个飞行特技表演",
+  "modifier.tinkersarchery.uplifting": "仰望",
+  "modifier.tinkersarchery.uplifting.description": "箭头稍微向上射出以帮助瞄准",
+  "modifier.tinkersarchery.uplifting.flavor": "举头望明月",
+  "modifier.tinkersarchery.weighty": "稳定",
+  "modifier.tinkersarchery.weighty.description": "使箭头更稳定，使其飞出更讨喜的轨迹",
+  "modifier.tinkersarchery.weighty.flavor": "芝麻稳定",

```

```suggestion
  "modifier.tinkersarchery.weighty": "稳定",
  "modifier.tinkersarchery.weighty.description": "使箭头更稳定，使其飞出更讨喜的轨迹",
  "modifier.tinkersarchery.weighty.flavor": "先稳后动",
```
只找到nilu以及Brandon Heath的歌曲